docker-compose.yaml

```diff
@@ -10,11 +10,19 @@ services:
       - SGX_SDK=/opt/sgxsdk
       - LD_LIBRARY_PATH=/opt/sgxsdk/sdk_libs
       - SGX_MODE=SW
-
-  # release:
-  #   devices:
-  #     - /dev/sgx/enclave
-  #     - /dev/sgx/provision
+  devhw:
+    build: .
+    stdin_open: true # docker run -i
+    tty: true        # docker run -t
```

Comment:
As an aside, how needed are these? (For the dev container, at least, things seem to work fine without this?)

Reply:
Not sure, feel free to change this at some point if it does not add value.